packages/components/checkbox/src/checkbox.ts

```diff
@@ -40,7 +40,7 @@ const Checkbox = defineComponent({
     const composedRefs = useComposedRefs(forwardedRef, buttonRef)
     const hasConsumerStoppedPropagationRef = ref(false)
     // We set this to true by default so that events bubble to forms without JS (SSR)
-    const isFormControl = computed(() => buttonRef.value ? Boolean(buttonRef.value?.closest('form')) : true)
+    const isFormControl = computed(() => buttonRef.value instanceof HTMLElement ? Boolean(buttonRef.value.closest('form')) : false)
```

Comment:
@productdevbook I accidentally changed this to false when I was fixing the vitest issue. Do you know why this happened when it is set to true?

vitest caught 1 unhandled error during the test run.
This might cause false positive tests. Resolve unhandled errors to make sure your tests are not affected.

buttonRef.value?.closest is not a function
 ❯ ReactiveEffect.fn packages/components/checkbox/src/checkbox.ts:43:85
     41|     const hasConsumerStoppedPropagationRef = ref(false)
     42|     // We set this to true by default so that events bubble to forms without JS (SSR)
     43|     const isFormControl = computed(() => buttonRef.value ? Boolean(buttonRef.value?.closest('form')) : tr…